app/src/components/main/scrollWrapper/scrollWrapper.scss

```diff
@@ -39,41 +39,60 @@
 .scrolling-content {
   z-index: 0;
 }
+.with-footer > .scrolling-content {
```

Comment:
Can such selectors be avoided? Why not just add corresponding class to the element?

Reply:
I can't add corresponding class directly to the element because it's encapsulated into `Scrollbars` element. I have no idea how to avoid this without editing `react-custom-scrollbars` library.

Reply:
I see that element with class `.scrolling-content` comes from default props (`scrollWrapper.jsx#73`)

Reply:
Really. Thanks! Fixed.